index.js

```diff
@@ -1,10 +1,8 @@
-var keypather = module.exports = function (opts) {
-  var keypather = new Keypather(opts && opts.force);
-  return keypather;
-};
+module.exports = Keypather;
 
-function Keypather (force) {
-  this.force = (force !== undefined) ? Boolean(force) : true; // force - default: true
+function Keypather (opts) {
+  if (!(this instanceof Keypather)) { return new Keypather(opts); }// Auto instantiate
+  this.force = (opts && opts.force && Boolean(opts.force) === false ? false : true); // force - default: true
```

Comment:
This should be:
`this.force = (opts && typeof opts.force !== 'undefined' && Boolean(opts.force) === false ? false : true);`
else it's always true.